.travis.yml

```diff
@@ -8,13 +8,13 @@ matrix:
   include:
   - os: linux
     dist: trusty # Ubuntu 14.04
-    dotnet: 2.1.300
+    dotnet: 5.0.408
```

Comment:
can you revert the changes to .travis.yml? it will clash with another PR, and also we're deprecating travis.